glossary.md

```diff
@@ -316,7 +316,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | reinforcement learning            | học tăng cường                            |                                              |
 | representation learning           | học biểu diễn                             | [https://git.io/JvojG](https://git.io/JvojG) |
 | rescale                           | chuyển đổi giá trị                        | [https://git.io/JfeXx](https://git.io/JfeXx) |
-| Residual Network                  | Mạng Phần dư                              |
+| Residual Network                  | Mạng Thặng dư                              |
 | reward function                   | hàm điểm thưởng                           |                                              |
 | robust (model, algorithm, system) | mạnh mẽ                                   | [https://git.io/Jfe1e](https://git.io/Jfe1e) |
```

Comment:
Chuyển đổi giá trị hiện mang nghĩa rộng hơn "rescale" - vốn dĩ là nâng giá trị lên với một tỷ lệ nào đó. Thường là đồng nhất cho cả dữ liệu. Còn chuyển đổi giá trị mang nghĩa chung và rộng hơn, dễ nhầm sang từ "transform". 

Từ robust dịch là "mạnh mẽ" cũng hơi gợn một chút vì có thể hiểu nhầm sang "chất lượng cao", trong khi ở đây là nó chịu được các thay đổi và ảnh hưởng như của nhiễu.

Reply:
nếu mô hình generalize tốt với 1 tập dữ liệu test hơi khác chút thì mình có nói nó `robust` không anh? Trước ai đó có nói từ `robust` chỉ mang nghĩa `kháng nhiễu` khi nó ở cụm `robust to noise`

Reply:
Em nhớ nhầm đoạn anh nói rồi. Cái khó là "nhiễu" trong tiếng Việt có thể hiểu theo nhiều nghĩa không chỉ mỗi "noise", và nó cũng có nghĩa chung chung. 
Để chỉ rõ ra thêm thì người ta nói "robust to cái gì đó". 

Còn như cái em viết ở trên, cái đó cũng có thể gọi là robust. Nhưng em chỉ có thể nói là "robust to ..." cái thay đổi gì mà em lập ra thôi. Vì nếu không test rộng ra, em bị "over claim" vì chưa chắc em đã tốt với sự thay đổi ở các tập dữ liệu khác. Nên khi dùng từ robust em cần đặt thêm ngữ cảnh cụ thể.

Reply:
@ngcthuong sorry em quên mất cái thread này
vậy trong trường hợp "robust to dataset shift" thì không dùng từ `kháng` được rồi, anh có đề xuất gì không ạ? `ổn định` thì sao anh?